src/test/java/sysc4806/graduateAdmissions/model/PrivilegeTest.java

```diff
@@ -20,17 +20,18 @@
 
     @BeforeEach
     public void setUp(){
-        privilege = Privilege.builder().build();
+        privilege = Privilege.builder().owner(Owner.SELF).operation(Operation.READ).target(Target.USER).build();
     }
 
     /**Test to ensure default values of fields of a Privilege are correct*/
     @Test
     public void testNoArgsConstructor(){
-        assertNotNull(privilege);
-        assertNull(privilege.getOperation());
-        assertNull(privilege.getOwner());
-        assertNull(privilege.getTarget());
-        assertEquals(0, privilege.getId());
+        Privilege noArgsPrivilege = new Privilege();
+        assertNotNull(noArgsPrivilege);
+        assertNull(noArgsPrivilege.getOperation());
+        assertNull(noArgsPrivilege.getOwner());
+        assertNull(noArgsPrivilege.getTarget());
+        assertNotNull(noArgsPrivilege.getId());
     }
 
```

Comment:
As an aside, we probably shouldn't be testing any of this because none of this is testing our code. It tests that Lombok is working correctly and that's not our domain. Same reason why we don't test that Spring works and the repository interfaces work.

No changes required